src/data/equiv/mul_add.lean

```diff
@@ -475,6 +483,9 @@ lemma mul_left_symm_apply (a : G) : ((equiv.mul_left a).symm : G → G) = (*) a
 lemma mul_left_symm (a : G) : (equiv.mul_left a).symm = equiv.mul_left a⁻¹ :=
 ext $ λ x, rfl
 
+@[to_additive]
+lemma mul_left_bijective (a : G) : function.bijective ((*) a) := (equiv.mul_left a).bijective
```

Comment:
These lemmas should not be in the `equiv` namespace.

Reply:
thanks!